docs/sdks/server-sdks/dotnet.md

```diff
@@ -3,39 +3,65 @@ import TabItem from '@theme/TabItem';
 
 # .NET
 
-Eppo's open source .NET SDK can be used for both feature flagging and experiment assignment:
+Eppo's open source .NET SDK can be used for feature flagging and experiment assignment and Multi-Armed Contextual Bandits:
```

Comment:
❤️ 

Any particular reason you capitalized it?

Reply:
In my haste, I wrote a little like little Winnie The Pooh, I suppose, where bandits are a Very Important Thing. oh bother